src/main/java/blackjack/gamer/Dealer.java

```diff
@@ -0,0 +1,62 @@
+package blackjack.gamer;
+
+import blackjack.card.CardDeck;
+import blackjack.card.Hand;
+import blackjack.result.BlackjackMatchResult;
+import blackjack.result.ProfitResult;
+import blackjack.state.started.finished.Stay;
+import java.math.BigDecimal;
+import java.util.LinkedHashMap;
+import java.util.List;
+import java.util.Map;
+
+public class Dealer extends Gamer {
+
+    private static final String NAME = "딜러";
+    private static final int HIT_THRESHOLD = 16;
+
+    public Dealer(Hand hand) {
+        super(hand);
+    }
+
+    private BlackjackMatchResult determineMatchResultFor(Player player) {
+        return getHand().determineMatchResultFor(player.getHand());
+    }
+
+    public ProfitResult calculateProfits(List<Player> players) {
+        BigDecimal dealerProfit = BigDecimal.valueOf(0);
```

Comment:
BigDecimal는 상수로 ZERO를 제공해주고있어요. 😃 

```suggestion
        BigDecimal dealerProfit = BigDecimal.ZERO;
```